EIPS/eip-918.md

```diff
@@ -1,27 +1,21 @@
 ---
 eip: 918
 title: Mineable Token Standard
+description: A specification for standardized Mineable Tokens that uses a Proof of Work algorithm for distribution.
 author: Jay Logelin <jlogelin@alumni.harvard.edu>, Infernal_toast <admin@0xbitcoin.org>, Michael Seiler <mgs33@cornell.edu>, Brandon Grill <bg2655@columbia.edu>
```

Comment:
Something odd is happening with this PR.  I see the latest commit that added a GitHub user to the author list, but I don't see that reflected here:
![image](https://user-images.githubusercontent.com/886059/156740949-65ecadc7-bc8b-46cc-9589-60e6ff9bef51.png)
Maybe this PR needs a rebase or cleanup or something?

Reply:
Rebased and changed username to `jlogelin` in the header. Still seeing the same errors, Please advise @MicahZoltu

Reply:
Rebased, changed username in header and still seeing the same error. Please advise @MicahZoltu 

```
---
## (fail) EIPS/eip-918.md
| classification |
| ------------- |
| `ambiguous` |
- eip-918.md has no identifiable authors who can approve the PR (only considering the base version)
Error: Process completed with exit code 1.
```